clusterloader2/pkg/measurement/util/latency_metric.go

```diff
@@ -54,3 +54,18 @@ func (metric *LatencyMetric) VerifyThreshold(threshold *LatencyMetric) error {
 	}
 	return nil
 }
+
+// ToPerfData converts latency metric to PerfData.
+func (metric *LatencyMetric) ToPerfData(name string, ratio float64) DataItem {
```

Comment:
This and extractLatencyMetrics to a separate PR please :)

Reply:
Moved to #623.